lib/devise_two_factorable/models/two_factorable.rb

```diff
@@ -119,7 +119,7 @@ def validate_otp_recovery_token(token)
     def validate_otp_token_with_drift(token)
       # should be centered around saved drift
       (-self.class.otp_drift_window..self.class.otp_drift_window).any? do |drift|
-        time_based_otp.verify(token, Time.now.ago(30 * drift))
+        time_based_otp.verify(token, drift_behind: (0.5 * drift), at: Time.now + (0.5 * drift))
```

Comment:
@bettinson Does this change anything you're working on in https://github.com/rietta-inc/devise-2fa/pull/14 ?

Reply:
@grepsedawk Should just be a simple refactor, not any functionality